projects/js-packages/critical-css-gen/changelog/add-separate-exports

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Add /node and /browser entrypoints.
```

Comment:
This could probably use updating now.

I might even add multiple files (e.g. `projects/js-packages/critical-css-gen/changelog/add-separate-exports#2` and so on), since there are a few different things going on here that might be worth mention in the changelog:

* Replacement of `BrowserInterfacePlaywright` with `BrowserInterfaceIframe` in the default entry point.
* Deletion of the `build-browser/bundle.js` and `build-browser/bundle.full.js` files.
* Addition of the `/playwright` entry point.
* Cleanup of exports, the package has always been ESM-only and this is now correctly reflected.

The first two are probably 'major' significance (although the second might be omitted if that was never really supposed to be "public"), and the third 'minor'. The fourth could be omitted.

Reply:
Oh yeah, I totally forgot about these 😬 Thanks.

I've updated the changelogs in 6b2846eaf41df09c24ab5eb8c54925decb8afdeb.